### PR TITLE
Remove "erasing" print from Directory.remove (Windows)

### LINKED ISSUE #39106: Directory.remove prints
Godot 3.2.2 beta3
Windows 10

When I use `Directory.remove`, this gets printed:
```
erasing <whatever path of the file I erase>
```

When erasing lots of files it spams the console. I think this should not print.
Apparently it only happens on Windows.

### PR DESCRIPTION
**Pull request by nathanfranke**
Fixes #39106

Haven't contributed in a while, nice simple bug to fix :)

### CODE REVIEW

**Comment by ThakeeNathees** [comment]
why don't replace it with `print_verbose()` ??

**Comment by nathanfranke** [comment]
Changed it to print verbose. Apologies for deleting the branch, I used underscores accidentally but it shouldn't matter.

File: drivers/windows/dir_access_windows.cpp [hunk 1]
```diff
@@ -286,7 +286,7 @@ Error DirAccessWindows::remove(String p_path) {
 
 	p_path = fix_path(p_path);
 
-	printf("erasing %s\n", p_path.utf8().get_data());
+	print_verbose("Erasing " + p_path.utf8().get_data());
```
Comment (akien-mga) [inline drivers/windows/dir_access_windows.cpp:289]: I think you can remove the print, `DirAccessUnix` doesn't print anything so it should be kept consistent. This was a debug print.

Comment (akien-mga) [reply drivers/windows/dir_access_windows.cpp:289]: Feel free to remove the dead code below too.

Comment (nathanfranke) [reply drivers/windows/dir_access_windows.cpp]: Fixed :smiley:

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**